### PR TITLE
fix(contribute): Creating a new user

### LINKED ISSUE #16966: Add instructions on how to create your first user
[CONTRIBUTING](https://github.com/freeCodeCamp/freeCodeCamp/blob/staging/CONTRIBUTING.md) currently describes how to set everything up, but leaves it up to the reader to understand that you will have to sign up locally, that the sign in link can be found in the console output or Mailhog and so forth.

It is not rocket science to figure it all out, but it would easier to get started if contributing documentation would show the whole flow.

Any takers?

### PR DESCRIPTION
**Pull request by stevenandersonz**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [x] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes #16966

#### Description
A brief explanation on how to create a new user.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -307,6 +307,15 @@ Run:
 docker-compose up
 ```
 
+
+### Creating a New User
```
Comment (ojongerius) [inline CONTRIBUTING.md]: Could you please move this section down, under the "Shared" header? Line 326 looks like a good spot.

**Review by ojongerius — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (stevenandersonz) [reply CONTRIBUTING.md]: Sure! I will do that.

Comment (ojongerius) [reply CONTRIBUTING.md]: 👍

**Review by ojongerius — APPROVED** [review]

**Comment by ojongerius** [comment]
Thanks for your contribution @stevenandersonz !

**Pull request merged**